software/contrib/harmonic_lfos.md

```diff
@@ -11,7 +11,7 @@ Six harmonically related sine/saw/square wave LFOs
     digital in: Reset all LFOs
     analogue in: Added to master rate
     knob 1: Master rate
-    knob 2: Adjusts the master clock division of the currently selected LFO
+    knob 2: Adjusts the master clock division of the currently selected LFO (or the spread amount for noise)
```

Comment:
[optional] I think it would be nice to be a little clearer on what 'spread' means. Maybe 'or the maximum voltage for noise'.